webpack.mix.js

```diff
@@ -12,12 +12,26 @@ const mix = require('laravel-mix').mix;
  */
 
 const resources = 'resources/assets';
```

Comment:
@puredazzle what do you think about adding a `theme` variable? Will make it easier to change the name.